src/Compiler/Service/SemanticClassification.fsi

```diff
@@ -49,7 +49,6 @@ type SemanticClassificationType =
     | TypeDef = 35
     | Plaintext = 36
 
-[<RequireQualifiedAccess>]
```

Comment:
`RequireQualifiedAccess` is defined as AttributeTargets.Class so it should be used here

Reply:
Reverted as we extended `RequireQualifiedAccess` to use `AttributeTargets.Struct`